<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [Failed suite [sample]]" guice-stage="DEVELOPMENT">
  <test thread-count="5" name="Datadriven(failed)(failed)">
    <classes>
      <class name="com.inetbanking.testcases.TC_loginDatadriven_002">
        <methods>
          <include name="setup"/>
          <include name="logindatadriven" invocation-numbers="0 1 2 3"/>
          <include name="teardown"/>
        </methods>
      </class> <!-- com.inetbanking.testcases.TC_loginDatadriven_002 -->
    </classes>
  </test> <!-- Datadriven(failed)(failed) -->
  <test thread-count="5" name="sample_verify(failed)(failed)">
    <classes>
      <class name="com.inetbanking.testcases.TC_Sample_Verify">
        <methods>
          <include name="tearDown"/>
          <include name="testVerifyTitle"/>
          <include name="testLogo"/>
          <include name="setUp"/>
        </methods>
      </class> <!-- com.inetbanking.testcases.TC_Sample_Verify -->
    </classes>
  </test> <!-- sample_verify(failed)(failed) -->
</suite> <!-- Failed suite [Failed suite [sample]] -->
